### PR TITLE
Closes #1336: Fire-TV Repackaging

### LINKED ISSUE #1336: Package Renaming
Part of refactoring, improves readability:

See https://github.com/mozilla-mobile/firefox-tv/issues/1085

### PR DESCRIPTION
**Pull request by psymoon**
### Pull Request checklist

- [x] This PR includes thorough **tests** or an explanation of why it does not
No need for test because it's part of refactoring

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -75,11 +75,11 @@
                 <data android:mimeType="text/plain" />
             </intent-filter>
         </activity>
-
-        <activity android:name=".webrender.InfoActivity"
+
```
Comment (liuche) [inline app/src/main/AndroidManifest.xml]: nitnit: leading trailing space

File: app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt [hunk 2]
```diff
@@ -116,9 +118,9 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
     }
 
     var onNavigationEvent: ((
-        event: NavigationEvent,
-        value: String?,
-        autocompleteResult: InlineAutocompleteEditText.AutocompleteResult?
+            event: NavigationEvent,
```
Comment (liuche) [inline app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt]: Is this indentation correct?

File: app/src/main/java/org/mozilla/tv/firefox/onboarding/OnboardingActivity.kt [hunk 3]
```diff
@@ -2,14 +2,15 @@
...
-package org.mozilla.tv.firefox
+package org.mozilla.tv.firefox.onboarding
 
 import android.app.Activity
 import android.content.Intent
 import android.os.Bundle
 import android.preference.PreferenceManager
 import android.support.v7.app.AppCompatActivity
 import kotlinx.android.synthetic.main.content_onboarding.*
+import org.mozilla.tv.firefox.R
```
Comment (liuche) [inline app/src/main/java/org/mozilla/tv/firefox/onboarding/OnboardingActivity.kt]: Necessary?

Comment (psymoon) [reply app/src/main/java/org/mozilla/tv/firefox/onboarding/OnboardingActivity.kt]: Yes

File: app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt (hunk 2 above)
Comment (psymoon) [reply app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt]: I think so? 8 line indentation as per ktlint suggestion

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (psymoon) [reply app/src/main/AndroidManifest.xml]: Outdated

**Comment by mcomella** [comment]
Was rebase and passes locally for Simon: merging.

**Pull request merged**